cinema/views.py

```diff
@@ -31,7 +46,7 @@ class CinemaHallViewSet(viewsets.ModelViewSet):
 
 
 class MovieViewSet(viewsets.ModelViewSet):
-    queryset = Movie.objects.all()
+    queryset = Movie.objects.all().prefetch_related("actors", "genres")
```

Comment:
`    queryset = Movie.objects.prefetch_related("actors", "genres")
`

Reply:
Not fixed